cost/cloud_cost_anomaly_alerts/README.md

```diff
@@ -1,8 +1,8 @@
-# Cloud Cost Anomaly Policy
+# Cloud Cost Anomaly Alerts Policy
```

Comment:
Should match the name in the top-level README 

Should be `Cloud Cost Anomaly Alerts` rather than `Cloud Cost Anomaly Alerts Policy`